pkgs/development/python-modules/jupyterlab-vim/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  hatchling,
+  hatch-jupyter-builder,
+  hatch-nodejs-version,
+  jupyterlab,
+}:
+
+buildPythonPackage rec {
+  pname = "jupyterlab-vim";
+  version = "4.1.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "jupyterlab_vim";
+    inherit version;
+    hash = "sha256-q/KJGq+zLwy5StmDIa5+vL4Mq+Uj042A1WnApQuFIlo=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  build-system = [
```

https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-parameters